internal/db/schema/migrations/oss/postgres/99/01_server_worker_release_version.up.sql

```diff
@@ -0,0 +1,53 @@
+begin;
+
+  alter table server_worker
+    add column release_version text;
```

Comment:
You may want to check in with @mgaffney around `not null` here -- I agree that we need to handle unspecified but there may be a preference for a sentinel value, which is already a type that exits.

Reply:
Thanks! I checked with Mike and he thinks null is appropriate for this instead of using a sentinel.